frontend/src/App.jsx

```diff
@@ -79,9 +79,10 @@ function App() {
           <Route path="/user/:id/update/recipe" element={<UpdateRecipe />} />
           <Route path="/recipe/:id" element={<OneRecipe />} />
           <Route path="/contributors" element={<Contributors />} />
+          <Route path="/privacy-policy" element={<LegalTerms />} />
           <Route path="*" element={<NotFound/>}/>
           {/* Add Contributors route */}
-          <Route path="/privacy-policy" element={<PrivacyPolicy />} />
+         
```

Comment:
Same comment as above

Reply:
@AlfiyaSiddique  ,  why do you want to revert these changes